### PR TITLE
updated readme.md

### LINKED ISSUE #69: Update Readme.md
update readme.md by adding installation steps, introduction and all.

### PR DESCRIPTION
**Pull request by animesh-007**
Updated Readme.md 
fixes #69

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,74 @@
...
+
+# officialWebsite
+
+[![Build Status](https://travis-ci.com/developer-student-club-thapar/officialWebsite.svg?branch=master)](https://travis-ci.com/github/developer-student-club-thapar/officialWebsite)  [![Slack Status](https://img.shields.io/badge/slack-chat-purple.svg?logo=slack)](https://dscthapar-gspatiala.slack.com/join/shared_invite/enQtNzU2MzA2MjcxNzkyLTkwNDRiNWMzYjUzYjNjYjM0M2JhMDgwOTI3MGQwYWU1NzNlNGMxZGVhNzk0MGZiYTI5YzgwZDhiMTk1MjE4M2M) [![GitHub issues](https://img.shields.io/github/issues/developer-student-club-thapar/officialWebsite?logo=github)](https://github.com/developer-student-club-thapar/officialWebsite/issues)
+</div>
+
+This is the repo for the official Website of DSC Thapar Institute of Engineering and Technology, Patiala , Punjab
 Head over to CONTRIBUTIONS.md for how to start contributing!
 
+Installation
```
Comment (shubhank-saxena) [inline README.md:15]: We don't want to combine our contributions and readme

File: README.md [hunk 2]
```diff
@@ -23,3 +89,11 @@ Thanks goes to these wonderful people for helping us set up this project. Cheers
 <!-- ALL-CONTRIBUTORS-LIST:END -->
```
Comment (shubhank-saxena) [inline README.md:89]: That table needs to come back. Rebase your master on this branch

**Comment by shubhank-saxena** [comment]
@animesh-007 I have sent a PR on your branch. Kindly merge as I have done some minor changes

**Comment by animesh-007** [comment]
> @animesh-007 I have sent a PR on your branch. Kindly merge as I have done some minor changes

Is there any change left to be done?

**Comment by shubhank-saxena** [comment]
> > @animesh-007 I have sent a PR on your branch. Kindly merge as I have done some minor changes
> 
> Is there any change left to be done?

Nopes. It looks good to me! Thanks for doing this! :rocket:

**Pull request merged**